src/Apps/ArtAdvisor/06-Discovery-Neartext/weaviate-db.ts

```diff
@@ -0,0 +1,264 @@
+import {
+  DiscoveryArtwork,
+  DiscoveryUser,
+  MongoID,
+  UUID,
+} from "Apps/ArtAdvisor/06-Discovery-Neartext/types"
+import _ from "lodash"
+import weaviate, { WeaviateClient, generateUuid5 } from "weaviate-ts-client"
+
+const DEFAULT_ARTWORKS_CLASS = "DiscoveryArtworks"
+const DEFAULT_USERS_CLASS = "DiscoveryUsers"
+
+export class WeaviateDB {
+  private client: WeaviateClient
+  private artworkClass: string
+  private userClass: string
+
+  constructor(
+    options: {
+      /** Weaviate host url */
+      url?: string
+      /** Name of artwork class to use.
+       * Be sure this class maps to the likes/dislikes of the
+       * user class that is being supplied here.
+       */
+      artworkClass?: string
+      /** Name of user class to use.
+       * Be sure this class's likes/dislikes are mapped to the
+       * same artwork class that is being supplied here.
+       */
+      userClass?: string
```

Comment:
Tried to parameterize this in case we wanted to quickly run these prototypes against a different dataset, but this is moot for now, since we decided today to recycle the same Weaviate collections.